yarn.lock

```diff
@@ -1451,20 +1471,20 @@ __metadata:
   peerDependencies:
     aws-cdk-lib: ^2.129.0
     constructs: ^10.3.0
-  checksum: d4e7563187cae816ae968bcf3758419762c520c9fa8c3a9ce8a93fb92c43067fdaf01783779d5ac201a6ed7abdf2716df6975b6c68111d7a6b15c8d8647401ae
+  checksum: 19a210a41319bdab35975a9de27f89082afcd4b6f6863a3551104063fe91c12d645dfded198e72d479c91c846259b4d381d641e5819528458b55883d05f56f53
   languageName: node
   linkType: hard
 
-"@aws-amplify/graphql-schema-generator@npm:0.9.3":
-  version: 0.9.3
-  resolution: "@aws-amplify/graphql-schema-generator@npm:0.9.3"
+"@aws-amplify/graphql-schema-generator@npm:^0.9.4":
+  version: 0.9.4
+  resolution: "@aws-amplify/graphql-schema-generator@npm:0.9.4"
   dependencies:
     "@aws-amplify/graphql-transformer-core": 2.9.3
     "@aws-amplify/graphql-transformer-interfaces": 3.10.1
-    "@aws-sdk/client-ec2": 3.338.0
-    "@aws-sdk/client-iam": 3.338.0
-    "@aws-sdk/client-lambda": 3.338.0
-    "@aws-sdk/client-rds": 3.338.0
+    "@aws-sdk/client-ec2": 3.624.0
+    "@aws-sdk/client-iam": 3.624.0
+    "@aws-sdk/client-lambda": 3.624.0
+    "@aws-sdk/client-rds": 3.624.0
```

Comment:
By looking at yarn.lock file it seems that category api updated V3 CDK.
But on the CLI side we depend on older.
And it seems that some transitive dependencies in yarn.lock file got duplicated as a result. Which is most likely why binary size grew that much.
We should try to align SDK version on CLI side with what data brings and see if this resolves size problem.

Reply:
Should I try to align all SDK version, not just the 4 deps you quoted above?

Reply:
yes. they all share common transitive dependencies. it looks like those got duplicated due to mismatches.

Reply:
All v3 SDKs to be precise.